spec/spec_helper.rb

```diff
@@ -103,7 +103,7 @@
   add_filter '/spec/'
 end
 
-SimpleCov.minimum_coverage 97
+SimpleCov.minimum_coverage 80
```

Comment:
Out of curiosity, what's the reason for lowering this? Is 80% the coverage for all of our projects?

Reply:
it's coz we're not doing any controller tests at the moment. we were relying on the features spec.